### PR TITLE
Refactored project structure to allow a simpler installation

### PR DESCRIPTION
**Pull request by devilicecream**
Put everything into a python module, so that it can be installed using the `setup.py` standard syntax. Plus, in this way the imports are not relative anymore, and that makes it simpler to avoid circular imports and allows a better structure. This is a starting point to solve #12

### CODE REVIEW

File: bombolone/__init__.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+__author__ = 'walter'
```
Comment (Zizzamia) [inline bombolone/__init__.py]: walter ??? I think you need change this one

File: bombolone/__init__.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import click
+from fabfile import minify, init_database, local_backup
+
+
+def bake():
+    """
+    Initialize the database from the backup and minify JS files to configure ("cook") the app.
```
Comment (Zizzamia) [inline bombolone/__init__.py]: I love this description :+1:

File: bombolone/__init__.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+COMMANDS = {
+    'bake': bake,
+    'serve': serve,
+    'refrigerate': refrigerate
+}
+
+
+@click.command()
+@click.argument('command')
+def main(command):
```
Comment (Zizzamia) [inline bombolone/__init__.py]: Very clean main

File: bombolone/api/account.py [hunk 4]
```diff
@@ -10,9 +10,9 @@
 from werkzeug import secure_filename
 
 # Imports inside Bombolone
-import core.users
-from core.utils import jsonify
-from decorators import authentication, get_hash, check_rank
+import bombolone.core.users
+from bombolone.core.utils import jsonify
+from bombolone.decorators import authentication, get_hash, check_rank
```
Comment (Zizzamia) [inline bombolone/api/account.py]: Nice!

File: bombolone/__init__.py (hunk 1 above)
Comment (devilicecream) [inline bombolone/__init__.py]: My bad. My IDE likes to give me all the ownership of my code xD

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,59 @@
+__author__ = 'walter'
```
Comment (Zizzamia) [inline setup.py]: Again, this need be change

**Comment by Zizzamia** [comment]
Well done! Amazing first contribution. #shipIt

**Pull request merged**